py3status/modules/online_status.py

```diff
@@ -50,17 +55,18 @@ def _connection_present(self):
                                    stdout=fnull, stderr=fnull) == 0
 
     def online_status(self):
-        response = {
-            'cached_until': self.py3.time_in(self.cache_timeout)
-        }
+        response = {'cached_until': self.py3.time_in(self.cache_timeout)}
```

Comment:
how does the value in line 58 get returned from the function?

what can happen to response defined in that line?  maybe it gets overwritten?